src/xavier/gui.py

```diff
@@ -283,9 +283,7 @@ def launch_gui(DEBUG=True):
                 genome=genome,
                 targets=values["-TARGETS-"]
                 if values["-TARGETS-"]
-                else xavier_base(
-                    "resources", "Agilent_SSv7_allExons_hg38.bed"
-                ),  # TODO should this be part of the genome config file?
+                else (""),  # TODO should this be part of the genome config file?
```

Comment:
If the above suggestion is accepted then this line can be deleted
```suggestion
```

Reply:
let me try and see if that works.